jobs/build-node-image.Jenkinsfile

```diff
@@ -71,16 +71,32 @@ lock(resource: "build-node-image") {
                                           ["--security-opt label=disable", "--mount-host-ca-certs", "--force"])
         }
 
-        stage("Push Manifest") {
-            pipeutils.push_manifest(arches, container_registry_staging_repo, container_registry_staging_image_tag, container_registry_staging_manifest_tag)
+        stage('Build Extension Container') {
+            pipeutils.build_remote_image(arches, commit, src_config_url, container_registry_staging_repo,
+                                         "${container_registry_staging_image_tag}-extensions", stream_info.yumrepo.file, false,
```

Comment:
One gotcha here that I hit when testing extensions building locally is that the extensions container currently needs the RHEL 9.4 repo as well: https://github.com/openshift/os/blob/26ecd2c01600e8b2532ea6edf6322c60dcac1846/extensions-ocp-rhel-9.6.yaml#L17

I think what we can do here is concatenate the `rhel-9.4.repo` and the `rhel-9.6.repo` repo into one file, and then pass that as the secret. Hmm, I guess to represent that in the pipecfg, maybe it should be a list of files that we concatenate instead of just one file?